poc_config/openmrs/apps/registration/app.json

```diff
@@ -7,15 +7,16 @@
                 { "name":"Estado Civil", "display":true }
             ],
             "autoCompleteFields":["familyName", "caste"],
-            "defaultIdentifierPrefix": "GAN",
-            "defaultVisitType": "OPD",
-            "searchByIdForwardUrl": "/patient/{{patientUuid}}",
             "showMiddleName": false,
             "encounterTypes" : {
```

Comment:
I think this is easier to maintain the values here, this is a config file, you can change anytime, but in `constants.js` you will need to rebuild and redeploy the app. Plus, here we're indicating and limiting the type of encounters that can be used in a particular module, registration module in this case.